Assets/HoloToolkit/Utilities/Scripts/ManualCameraControl.cs

```diff
@@ -28,6 +28,18 @@ public enum MouseButton
     public MouseButton MouseLookButton = MouseButton.Shift;
     public bool IsControllerLookInverted = true;
 
+    public enum ControlMode
```

Comment:
ManualCameraControl should be moved to the HoloToolKit namespace so that we can move this enum outside the class scope and just rename the member back to just be ControlMode
Happy to do that now or it could be done in a seperate step